src/controller/controller.cpp

```diff
@@ -28,6 +28,7 @@ long Controller::run() {
 
 bool Controller::simulate() {
     terminalview.render(world);
+    if(sdlview.render(world)) return false;
```

Comment:
This is confusing.
I think it should return false if `sdlview.render()` returns false, meaning you should let .render() return false on quit

Reply:
It was a late Friday evening but I think I thought myself something when doing it.  
I'll look into it tho

Reply:
3f4a19d103d8272eba6bc22740cedf781d40af86